### PR TITLE
Minor source page updates

### PR DESCRIPTION
**Pull request by kmshin1397**
- Remove redundant form title in source notifications
- Fix seed telescopes list to have nicknames and names properly assigned (they were flipped)
- Move surveys list to above photometry

[CH #1317](https://app.clubhouse.io/astromarshal/story/1317/remove-send-a-notification-from-source-notification-accordion)
[CH #1315](https://app.clubhouse.io/astromarshal/story/1315/move-surveys-above-photometry)
[CH #1324](https://app.clubhouse.io/astromarshal/story/1324/display-nicknames-for-groups-on-scanning-page)

### CODE REVIEW

**Comment by kmshin1397** [comment]
Actually, don't merge yet - realized I have a couple more stories I could tack on to this for a general Source page update PR

**Review by stefanv — APPROVED** [review]

**Comment by kmshin1397** [comment]
Ready for review now

**Review by acrellin — APPROVED** [review]
Will merge when green

**Pull request merged**
